draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -373,22 +373,49 @@ using a right arrow to separate inputs from outputs: f : inputs → outputs.
 The generic hybrid PQ/T KEM frameworks we define depend on the the following
 cryptographic primitives:
 
+- Pseudorandom Generators ({{prgs}})
 - Key Encapsulation Mechanisms ({{kems}})
 - Nominal Groups ({{groups}})
-- Pseudorandom Generators ({{prgs}})
 - Key Derivation Functions ({{kdfs}})
 
 In the remainder of this section, we describe functional aspects of these
 mechanisms.  The security properties we require in order for the resulting
 hybrid KEM to be secure are discussed in {{security}}.
 
+## Pseudorandom Generators {#prgs}
+
+A pseudorandom generator (PRG) is a deterministic function `G` whose outputs
+are longer than its inputs. When the input to `G` is chosen uniformly at
+random, this induces a certain distribution over the possible output. The
+output distribution is pseudorandom if it is indistinguishable from the
+uniform distribution.
+
+- `New(seed) -> state`: Produce a new PRG using `seed` as the initial entropy.
+- `Read(state, n) -> out`: A deterministic algorithm to generate an `n`-byte
+  string of pseudo-random bytes.  The `state` value may be mutated by this
+  function.
```

Comment:
I'd be careful with new APIs introduced now, the PRGs used in our concrete instances don't use this API (SHAKE), they just have a one shot invocation; who needs a multi-step API? @bifurcation